x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.test.tsx

```diff
@@ -212,16 +212,17 @@ describe('Body', () => {
 
     test('it dispatches the `REMOVE_COLUMN` action when there is a field removed from the custom fields', async () => {
       const customFieldId = 'my.custom.runtimeField';
-      const extraFieldProps = {
-        ...props,
-        columnHeaders: [
-          ...defaultHeaders,
-          { id: customFieldId, category: 'my' } as ColumnHeaderOptions,
-        ],
-      };
+      const { storage } = createSecuritySolutionStorageMock();
+      const state = set<State>(cloneDeep(mockGlobalState), 'timeline.timelineById.timeline-test', {
```

Comment:
Instead of clone deep which _may_ slow down the tests, can we just make a factory function that returns the mockGlobalState? And we can remove `set` and just use the spread operator?